docs/changelog.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## v0.9.0 (30/11/2020)
+
+**Highlights**
+
+1. Support more human pose estimation methods.
+    - [MSPN](https://arxiv.org/abs/1901.00148)
+    - [RSN](https://arxiv.org/abs/2003.04030)
+2. Support video pose estimation datasets.
+    - [sub-JHMDB](http://jhmdb.is.tue.mpg.de/dataset)
+3. Support Onnx model conversion
+
+**New Features**
+- Support MSPN ([#278](https://github.com/open-mmlab/mmpose/pull/278))
+- Support RSN ([#221](https://github.com/open-mmlab/mmpose/pull/221), [#318](https://github.com/open-mmlab/mmpose/pull/318))
+- Support new post-processing method for MSPN & RSN ([#288](https://github.com/open-mmlab/mmpose/pull/288))
+- Support sub-JHMDB dataset ([#292]((https://github.com/open-mmlab/mmpose/pull/292)))
+- Support urls for pre-trained models in config files. ([#232](https://github.com/open-mmlab/mmpose/pull/232))
+- Support Onnx ([#305](https://github.com/open-mmlab/mmpose/pull/305))
+
+**Bug Fixes**
+- Fix model download links in README ([#255](https://github.com/open-mmlab/mmpose/pull/255), [#315](https://github.com/open-mmlab/mmpose/pull/315))
+
+**Deprecated**
+- Add deprecation warnings for `post_process=True/False` and `unbiased_decoding=True/False` ([#288](https://github.com/open-mmlab/mmpose/pull/288)).
```

Comment:
`post_process=True/False` and `unbiased_decoding=True/False` are deprecated, use `post_process=None|default|unbiased` etc. instead.